source/ceylon/time/Duration.ceylon

```diff
@@ -35,6 +45,14 @@ shared serializable class Duration(milliseconds) satisfies ReadableDuration & Sc
         value result = 3;
         return prime * result + milliseconds.hash;
     }
+    
+    shared actual Comparison compare(Duration other) => milliseconds.compare(other.milliseconds);
+    
+    shared actual Duration plus(Duration other) => Duration(milliseconds.plus(other.milliseconds));
+    
+    shared actual Duration negated => Duration(milliseconds.negated);
+    
+    shared actual Duration minus(Duration other) => Duration(milliseconds.minus(other.milliseconds));
```

Comment:
Wouldn’t this be simpler to read:

    milliseconds - other.milliseconds

Reply:
I wanted to make it visible that it just delegates to the corresponding method on the milliseconds, but I won't object changing it if you like. Should I?